SpeakUp/ExtraGrammarUtility.cs

```diff
@@ -116,12 +116,13 @@ public static void ExtraRulesForPawn(string symbol, Pawn pawn, Pawn other = null
             }
 
             //childhood
-            MakeRule(symbol + "childhood", pawn.story.childhood.identifier);
+            if (pawn.story.childhood != null) MakeRule(symbol + "childhood", pawn.story.childhood.identifier);
```

Comment:
The MakeRule method already has a built-in null check. That's pretty much all it does, actually. Where there any errors?

Reply:
We have this check on the adulthood because there could be instances where a pawn does not have an adulthood. There was a user who reported errors on the discord, and he was using mods that allowed him to have children, which were not given childhoods at all, and the error stopped when he gave them a childhood. The check here is more for other mod support, as sometimes mods may give pawns no childhood for whatever reason. Is the null check on adulthood not needed as well?

Reply:
The null check on adulthood predates the MakeRule function. But if there have been errors, then they must happen even before the method is called for. There is a way to simplify this, anyway... I'll merge and then modifiy it.